client/my-sites/upgrades/domain-management/email-forwarding/email-forwarding-details.jsx

```diff
@@ -7,18 +7,18 @@ var React = require( 'react' );
  * Internal dependencies
  */
 var i18n = require( 'lib/mixins/i18n' ),
+	support = require( 'lib/support' ),
 	analyticsMixin = require( 'lib/mixins/analytics' );
 
 var EmailForwardingDetails = React.createClass( {
 	mixins: [ analyticsMixin( 'domainManagement', 'emailForwarding' ) ],
 
 	render: function() {
-		var supportLink = 'https://' + i18n.getLocaleSlug() + '.support.wordpress.com/email-forwarding/';
```

Comment:
I'm not sure how to handle these, as far as I know we shouldn't prefix with the locale ever.